docs/quickstarts/packet.md

```diff
@@ -141,126 +124,112 @@ ssh-add ~/.ssh/id_rsa
 ssh-add -L
 ```
 
-Run the following command to create the cluster:
+Deploy the cluster:
 
 ```console
 lokoctl cluster apply
 ```
 
-Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
-under `asset_dir`.
-
-## Verification
-
-A successful installation results in the output:
+The deployment process may take a couple of minutes. Upon successful completion, an output similar
```

Comment:
I'd be more explicit in how much time it will take. In other platforms (what users might be more used to) it takes way less. I'd explicitly mention that it takes in the order of 10 minutes, to avoid frustration on taking more that the user might expect.

Reply:
Done.